string[] types
---
semap_msgs/ObjectDescription[] descriptions
